src/CMakeLists.txt

```diff
@@ -664,6 +667,12 @@ if(NOT WIN32 AND NOT APPLE)
     endif()
 endif()
 
+############################################################
+# MIOpen depends on libMLIRMIOpen for igemm kernels
+if(LIBMLIRMIOPEN AND NOT MIOPEN_USE_COMGR)
```

Comment:
I validated manually such that this won't pass Hip Debug COMGR stage:
 - That stage has hard coded flag: `-DMIOPEN_USE_COMGR=On`
 - If I also link in the libMLIRMIOpen.a, it ended up fail for solvers like ConvHipImplicitGemmMlirCppFwd. The reason of failure has nothing to do with linking, but simply that COMGR code path isn't done (cannot find the kernel in the statically populated `miopen_kernels.h` and `miopen_kernel_includes.h`).

Since we have decided that this PR only work for the offline build path, I'm marking this as resolved.

Reply:
Resolved.